tests/baselines/reference/user/github.log

```diff
@@ -0,0 +1,8 @@
+Exit Code: 1
+Standard output:
+index.ts(1,25): error TS7016: Could not find a declaration file for module 'github'. '/home/nathansa/ts/tests/cases/user/github/node_modules/github/index.js' implicitly has an 'any' type.
```

Comment:
so what is the test testing then?

Reply:
That's not supposed to be part of the test -- github moved to `@octokit/rest`. I'll go ahead and move it as part of this PR.